### DISCUSSION TITLE
BUG: Email address configured in Options resets to blank

### DISCUSSION THREAD

**Discussion by gitcodehelp** [sheltermanager/asm3#1480] [Q&A / Help] [answered]
To reproduce:
1) Settings > Options > Email tab > set "Email address" = address@email.com > Save 
3) Go to any other Options tab, change an option (say, "Home Page" tab, uncheck "Show tips...") > Save
4) Return to Options -> Email. "Email address" has been reset to blank.

This seems to happen regardless of the "from_address" setting in /etc/asm3.conf (i.e., whether it is commented out,
or same as/different from the address configured in Options).

ASM v48 (release)

**Comment by bobintetley** [comment, marked as answer]
Thank you for the bug report. It looks like the email address in one of the person choosers was masking the email address on subsequent saves. I've applied a fix now, commit 6798a48d7c19bf5e7d747b302f1c8aed4836177b

**Reply by gitcodehelp** [reply]
Thank you!
